common/djangoapps/student/models.py

```diff
@@ -2233,18 +2258,30 @@ def __unicode__(self):
 #### Helper methods for use from python manage.py shell and other classes.
 
 
+def strip_if_string(value):
+    if isinstance(value, basestring):
```

Comment:
Nit: This is fine, but as we move toward Py3 we should probably try to do these tests instead to reduce compatibility issues:
`if isinstance(value, six.string_types):`